$NetBSD: patch-configure,v 1.1 2026/03/23 10:47:18 nia Exp $

Make sure the test for "ANSI C headers" actually conforms to ANSI C.

Fixes build with modern GCC.

--- configure.orig	2026-03-23 10:24:05.009456817 +0000
+++ configure
@@ -5126,6 +5126,7 @@ _ACEOF
 cat confdefs.h >>conftest.$ac_ext
 cat >>conftest.$ac_ext <<_ACEOF
 /* end confdefs.h.  */
+#include <stdlib.h>
 #include <ctype.h>
 #if ((' ' & 0x0FF) == 0x020)
 # define ISLOWER(c) ('a' <= (c) && (c) <= 'z')
